src/components/chip/examples/chip-link.tsx

```diff
@@ -0,0 +1,42 @@
+import { Component, h } from '@stencil/core';
+
+/**
+ * Chip as hyperlink
+ * A chip can be used to navigate to a different page, pane of the application
+ * or to an external site. To get the chip to act as a hyperlink, you need to
+ * provide a `Link`.
+ *
+ * This way, the component will automatically become a native HTML link.
+ * This gives the users the possibility of interacting with links
+ * in a natural way, for instance they can open any of the previous
+ * pages in a new browser tab. This can also have other accessibility benefits.
+ *
+ * :::note
+ * Clicking a link will open it in current window by default,
+ * and this reloads the entire webpage.
+ * To avoid reloading the whole application (in the context of a single-page apps),
+ * you might want to handle the navigation with your application's router,
+ * :::
+ */
+@Component({
+    tag: 'limel-example-chip-link',
+    shadow: true,
+})
+export class ChipLinkExample {
+    link = {
+        href: 'https://github.com',
+        title: 'Open Github',
+        target: '_blank',
+    };
+
+    public render() {
+        return (
+            <limel-chip
+                id="1"
+                text="Github"
+                icon="github_copyrighted"
+                link={this.link}
```

Comment:
We don't use the `target` property from the `Link`. I think we should 😊

Reply:
⚡